ebuild-maintenance/removal/text.xml

```diff
@@ -113,17 +113,12 @@ In order to remove a virtual package, follow the following procedure:
     Wait the time appropriate for the last rites.
   </li>
   <li>
-    Update all ebuilds not to reference the virtual. Since there is
-    no urgent need to remove the virtual from user systems
-    and the resulting rebuilds would be unnecessary, do not bump ebuilds
-    when replacing the dependency.
+    Update all ebuilds not to reference the virtual, following normal
+    <uri link="::general-concepts/ebuild-revisions/"/> policy
```

Comment:
As I said, you may want to drop the last bullet point from the [target page](https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html), which refers back to this section and says:

> Examples of changes that can be done without a revision bump are:
> - [...]
> - a dependency change due to removal of a virtual package.

Reply:
I interpreted your comment to mean "the last bullet point in the overall list", as you can see in the diff: https://github.com/gentoo/devmanual/compare/3d8fdf443d4dfc945b48b93bb1451d6bfa8be403..d9683815c7a740c30b5d668b778f40de25a8864f. I'll adjust it later.